--- table_schema_for_repro.sql	2023-08-31 04:44:55.438589724 +0000
+++ table_schema_for_repro_copy.sql	2023-08-31 04:53:12.714570483 +0000
@@ -1,11 +1,11 @@
 CREATE TABLE public.bug_repro_values (
     id bigint NOT NULL,
+    condition_value integer NOT NULL,
     condition_category text NOT NULL,
-    condition_value integer NOT NULL
-    CONSTRAINT contrived_condition_check CHECK (
-CASE
-    WHEN (condition_category = 'must_be_positive') THEN (condition_value > 0)
-    WHEN (condition_category = 'must_be_negative') THEN (condition_value < 0)
-    ELSE true
+    CONSTRAINT contrived_condition_check CHECK (,
+CASE,
+    WHEN (condition_category = 'must_be_positive') THEN (condition_value > 0),
+    WHEN (condition_category = 'must_be_negative') THEN (condition_value < 0),
+    ELSE true,
 END)
 );
